### DISCUSSION TITLE
sd maid 2 README

### DISCUSSION THREAD

**Discussion by hui26** [d4rken-org/sdmaid-se#736] [General]
someone access my samsung phone remotely and hijacked my google account, and now i also can't access play store app. i also can't upgrade to pro, it says unavailable.i think they save anything downloaded on network storage links to my file samsung app, include record debug log.
can you help me pls?

**Comment by d4rken** [comment]
No one has hacked you. If you get "Google Play unavailable" just try a reboot of your device, it's a random bug in Google Play.
